octoprint_Spoolman/templates/Spoolman_sidebar.jinja2

```diff
@@ -113,7 +113,10 @@
                             </div>
                         <!-- /ko -->
 
-                        <div>
+                        <div style="text-align: center;">
+                            <span
```

Comment:
As spool id is dynamic in length, ids longer than just 1 character seem to shift the layout of spool row display. As much as I would love to prevent too much vertical space being wasted, I believe it will be better to move it to the "column" with the text, and display it eg. below all the current data, in a separate line.

Reply:
I had not noticed that, but after adding a couple hundred spools to my test instance I see what you mean, I will explore this a bit more in the new PR.